src/segment.rs

```diff
@@ -59,6 +60,7 @@ impl Segment {
 
     #[inline]
     pub fn set_best_segment_time(&mut self, time: Time) {
+        self.comparisons.insert(BEST_SEGMENT_COMPARISON_NAME.into(), time);
```

Comment:
well, I'd rather have it not working fully atm, than introducing code that makes it temporarily work and cause bugs later on if we forget to remove it.